hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/ozoneimpl/TestOnDemandContainerDataScanner.java

```diff
@@ -96,14 +99,13 @@ public void testUnscannedContainerIsScanned() throws Exception {
 
   @AfterEach
   public void tearDown() {
-    OnDemandContainerDataScanner.shutdown();
+    onDemandScanner.shutdown();
   }
 
   @Test
   public void testScanTimestampUpdated() throws Exception {
-    OnDemandContainerDataScanner.init(conf, controller);
     Optional<Future<?>> scanFuture =
-        OnDemandContainerDataScanner.scanContainer(healthy);
+        onDemandScanner.scanContainer(healthy);
```

Comment:
These can be moved to line 107. There are other line similar these below, We can move them to the line above.